tests/util/test_data_catalog_helper.py

```diff
@@ -0,0 +1,124 @@
+# MIT License
+#
+# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+"""
+Unit tests for the cray_product_catalog.util.catalog_data_helper module
+"""
+
+import unittest
+from tests.mocks import (
+  YAML_DATA, YAML_DATA_MISSING_MAIN_DATA,
+  YAML_DATA_MISSING_PROD_CM_DATA,
+  MAIN_CM_DATA, PROD_CM_DATA
+)
+import yaml
+from typing import Dict

```

Comment:
Standard modules should be imported first.
```suggestion
import yaml
from typing import Dict
from tests.mocks import (
  YAML_DATA, YAML_DATA_MISSING_MAIN_DATA,
  YAML_DATA_MISSING_PROD_CM_DATA,
  MAIN_CM_DATA, PROD_CM_DATA
)
```